src/api/app/models/event/relationship_create.rb

```diff
@@ -2,6 +2,8 @@ module Event
   class RelationshipCreate < Relationship
     self.message_bus_routing_key = 'relationship.create'
     self.description = 'Relationship was created'
+
+    receiver_roles :maintainer, :bugowner, :reviewer, :downloader, :reader
```

Comment:
We realized that this receiver_roles makes no sense at all. We're gonna remove it, but we need to change then the way we look for the subscribers in the `EventSubscription::FindForEvent#subscriptions`